src/payment/strategies/paypal-commerce/paypal-commerce-payment-processor.ts

```diff
@@ -0,0 +1,62 @@
+import { MissingDataError, MissingDataErrorType } from '../../../common/error/errors';
+
+import { OverlayPaypal } from './index';
+
+const modalWidth = 450;
```

Comment:
is there a way to customize this? Have you tested how this works in mobile?

Reply:
In mobile window open in new tab. And it's okay for this case